vital/algo/algorithm.h

```diff
@@ -202,20 +182,18 @@ class VITAL_ALGO_EXPORT algorithm
 // ------------------------------------------------------------------
 /// An intermediate templated base class for algorithm definition
 
-/**
- *  Uses the curiously recurring template pattern (CRTP) to declare the
- *  clone function and automatically provide functions to register
- *  algorithm, and create new instance by name.
- *  Each algorithm definition should be declared as shown below
- *  \code
+/// Uses the curiously recurring template pattern (CRTP) to declare the
+/// clone function and automatically provide functions to register
+/// algorithm, and create new instance by name.
+/// Each algorithm definition should be declared as shown below
+/// \code
     class my_algo_def
     : public algorithm_def<my_algo_def>
     {
       ...
     };
-    \endcode
```

Comment:
I had to manually remove the comments on the lines between the `\code` and `\endcode`.  Given that this happened only once in ~100 files I thought it was a fair trade.